### PR TITLE
organize pages into different classes; bump version

### PR DESCRIPTION
**Pull request by ApolloBian**
- Organize pages into Functions, Classes and Modules
 - update to TensorFlow 1.8.0

### CODE REVIEW

File: dashing.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
 {
-    "name": "TensorFlow",
+    "name": "TensorFlow 1.8.0",
```
Comment (ppwwyyxx) [inline dashing.json:2]: The name cannot be changed.

File: preprocess.sh [hunk 2]
```diff
@@ -5,8 +5,8 @@
 [[ -z "$1" ]] && exit 1
 
 set +e
-rm -rf "$1"/versions/master/api_docs/python/tf/contrib/keras*
-rm -rf "$1"/versions/master/api_docs/python/tf/keras
+# rm -rf "$1"/versions/master/api_docs/python/tf/contrib/keras*
+# rm -rf "$1"/versions/master/api_docs/python/tf/keras
```
Comment (ppwwyyxx) [inline preprocess.sh:9]: Including Keras results in pages with duplicated names but different cases, which contradicts with dash's docset standard.

File: transform.py [hunk 3]
```diff
@@ -39,27 +42,41 @@ def remove(*args, **kwargs):
 
 remove('header')
 remove('footer')
-remove('div', attrs={'class': 'devsite-nav-responsive-sidebar-panel'})
```
Comment (ppwwyyxx) [inline transform.py:42]: Why are these lines removed?

File: transform.py [hunk 4]
```diff
@@ -39,27 +42,41 @@ def remove(*args, **kwargs):
...
+#     for k in allcss[1:]:
+#         k.extract()
+
+# # filter buggy title
+# h4s = soup.findAll('h4')
+# if h4s:
+#     for h4 in h4s:
+#         if '{:#' in h4.text:
+#             code = h4.findAll('code')[0]
+#             h4.contents = [code]
```
Comment (ppwwyyxx) [inline transform.py:79]: Why are these lines removed?

File: transform.py (hunk 3 above)
Comment (ApolloBian) [reply transform.py:42]: I choose to only remove header, footer and left side bar.
I keep the right navigation bar for quick navigation inside page. (It was removed because it's `<nav>`)

File: preprocess.sh (hunk 2 above)
Comment (ApolloBian) [reply preprocess.sh:9]: I didn't run into any problems keeping `tf.keras` and `tf.contrib.keras`

**Comment by ppwwyyxx** [comment]
If you could send the PR that only does one thing "organize pages into different classes", it would be very much appreciated. 
The PR now contains a lot of personal preferences that should better be kept private. The docset is part of dash-user-contributions, so some sort of consistency (including the name at the very least) has to be preserved. It also removes some working code which I'm not sure why -- maybe some of the code is invalid now, but it'll need to be clarified and addressed differently.

**Comment by ApolloBian** [comment]
Sure.I didn't think about consistency issue at the beginning, since I was using it for my own. I have been a bit occupied with other stuff recently. I will try to make a new PR next month.
